.travis.yml

```diff
@@ -6,13 +6,14 @@ os:
 cache: cargo
 sudo: false
 script:
-  - cargo test
-  - pushd plugin
-  - cargo test
-  - popd
-  - pushd runner
-  - cargo install -f
-  - popd
-  - pushd examples/example_project
-  - cargo mutagen
-  - cargo mutagen -- --coverage
+  - cargo build
+  - cargo test -p mutagen-core # run unit tests first
+  - cargo test -p mutagen-selftest # run integration tests
+  - cargo test # run all (other) tests
+
+  # run executable cargo-mutagen on test project
+  - pushd examples/simple
+  - cargo run --bin cargo-mutagen --package cargo-mutagen
+
+  # check format
+  cargo fmt -- --check
```

Comment:
Is this a good idea? I've seen this in a blogpost. However, I also seen arguments against this.

Reply:
I'm not opposed to it. Let's leave it in and see if it burdens us too much.